bedrock/firefox/templates/firefox/testflight.html

```diff
@@ -0,0 +1,125 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title_prefix %}{{ _('Firefox for iOS') }} — {% endblock %}
+{% block page_title %}{{ _('TestFlight') }}{% endblock %}
+{% block page_desc %}{{ _('Sign up to test pre-release versions of Firefox for iOS via Apple’s TestFlight Beta program and help make our mobile browser for iPhone, iPad and iPod touch even better.') }}{% endblock %}
+
+{% block body_id %}firefox-ios-testflight{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_ios_testflight' %}
+{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block content %}
+<section id="intro">
+  <div class="container">
+    {# No more new logos! #}
+    {{ high_res_img('firefox/desktop/tips/firefox-logo.png', {'alt': 'Firefox for iOS', 'width': '136', 'height': '142', 'id': 'header-logo' }) }}
```

Comment:
Yay to not adding any more logos 👍 

Rather than piling on the `desktop/tips` directory though, would it be better to move the existing logo to a common directory?